lib/rps.rb

```diff
@@ -0,0 +1,25 @@
+class RPS
+
+  attr_reader :player, :opponent
+
+  def initialize(player, opponent)
+    @player = player.downcase
+    @opponent = opponent
+  end
+
+  def result
+    return "draw" if draw? == true
```

Comment:
I think this could be refactored one step further to:
return "draw" unless  !@draw?